packages/utils/src/lib/reports/generate-md-report.ts

```diff
@@ -40,13 +44,17 @@ export function generateMdReport(
   report: ScoredReport,
   options?: MdReportOptions,
 ): string {
+  const { isScoreListed, ...opts } = options ?? {};
   return new MarkdownDocument()
     .heading(HIERARCHY.level_1, REPORT_HEADLINE_TEXT)
     .$concat(
       ...(hasCategories(report)
-        ? [categoriesOverviewSection(report), categoriesDetailsSection(report)]
+        ? [
+            categoriesOverviewSection(report, { isScoreListed }),
+            categoriesDetailsSection(report, { isScoreListed }),
+          ]
         : []),
-      auditsSection(report, options),
+      auditsSection(report, { isScoreListed, ...opts }),
```

Comment:
The `isScoreListed` destructuring is needlessly verbose and unnecessary in this function, every usage re-structures it into an object again. Just passing down the `options` would be simpler:

![image](https://github.com/user-attachments/assets/d1ebf634-4040-40a4-80c6-1e8fa4a346ae)